tests/unit_test/lighter/participant_test.py

```diff
@@ -45,7 +45,7 @@ def test_invalid_org(self, invalid_org):
 
     @pytest.mark.parametrize(
         "invalid_type",
-        [("type1"), ("type@"), ("type!"), ("type~"), ("gateway"), ("firewall")],
+        [("type@"), ("type!"), ("type~")],
```

Comment:
so we allow ("gateway"), ("firewall") as type right now?

Reply:
Yes. This is a request from some customers so that they can use our Provision tool to provision their participant types. And this is one of the main purposes of this PR.